core/modules/css/CssAccess.cc

```diff
@@ -503,7 +503,8 @@ CssAccess::getPartTableParams(std::string const& dbName,
 
     std::vector<std::string> subKeys{"partitioning", "partitioning/subChunks",
         "partitioning/dirDb", "partitioning/dirTable", "partitioning/dirColName", "partitioning/latColName",
-        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName"};
+        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName",
+        "partitioning/lockInMem", "partitioning/scanSpeed"};
```

Comment:
Hmm, now I am confused, sorry :). Confused by your comment "I'm still a bit unsure that adding shared scan parameters to PartTableParams struct is the way to go". I thought that is what you suggested when we talked on Wednesday. (My original idea was to have a dedicated struct for shared scan parameters that'd be like PartTableParams or MatchTableParams, and we'd add it to TableParams). And now you are suggesting moving to a dedicated prefix for shared scans parameters instead of "partitioning", putting config values in their own namespace, and now you are unsure if they should even be in PartTableParams. Are you saying I should try going with my original idea?